### PR TITLE
[zh] fix document description problem

### PR DESCRIPTION
**Pull request by 0xff-dev**
Perhaps `EndpointSlices` should be used as a noun and should not be translated separately.
[service-networking/service/#endpointslice](https://kubernetes.io/zh/docs/concepts/services-networking/service/#endpointslice)

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -345,12 +345,12 @@ endpoints.
 Endpoint Slices provide additional attributes and functionality which is
 described in detail in [EndpointSlices](/docs/concepts/services-networking/endpoint-slices/).
 -->
-Endpoint 切片是一种 API 资源，可以为 Endpoint 提供更可扩展的替代方案。
-尽管从概念上讲与 Endpoint 非常相似，但 Endpoint 切片允许跨多个资源分布网络端点。
-默认情况下，一旦到达100个 Endpoint，该 Endpoint 切片将被视为“已满”，
-届时将创建其他 Endpoint 切片来存储任何其他 Endpoint。
+EndpointSlices 是一种 API 资源，可以为 Endpoints 提供更可扩展的替代方案。
+尽管从概念上讲与 Endpoints 非常相似，但 EndpointSlices 允许跨多个资源分布网络端点。
+默认情况下，一旦到达100个 Endpoint，该 EndpointSlice 将被视为“已满”，
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/service.md:350]:
```suggestion
默认情况下，一旦到达 100 个 Endpoint，该 EndpointSlice 将被视为“已满”，
```

Comment (0xff-dev) [reply content/zh/docs/concepts/services-networking/service.md]: Thank you. It has been corrected

**Comment by 0xff-dev** [comment]
I signed it

**Comment by 0xff-dev** [comment]
I signed it

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**